internal/testing/alldeps

```diff
@@ -29,6 +30,7 @@ github.com/google/subcommands
 github.com/google/uuid
 github.com/google/wire
 github.com/googleapis/gax-go
+github.com/googleapis/gax-go/v2
```

Comment:
Is this mongo's dependency?

Reply:
Oops, that's because I ran the script on Go 1.12. Removed.